### ISSUE TITLE
Regarding the Help button and foreign language content

### ISSUE THREAD

**Issue by GaurangTandon** [ChallengeHunt/challengehunt#4]
Hello there!

First of all, it's an excellent extension and must have for someone who likes taking part in contests! I have two suggestions:

1) The Help button (in the popup) should be at the top, perhaps below the logo. Currently, it requires me to scroll all the way to the bottom. It might not get noticed by many users.
2) Challenges of foreign languages are often shown in the popup, which I am unable to solve. Hence, allow me to set a filter (a simple checkbox might do) to remove all such contests from being shown.

By the way, I noticed that you are using many libraries (as the Github page lists). I recommend you try reducing your app's dependency on them. 697KB for an extension like this is too large imho.

I hope I made myself clear!

**Comment by architv** [comment]
Hello @GaurangTandon !

We are so glad that you found it useful!
1. That's a reasonable request. We would consider doing that.
2. There's a text box below the 'Challenge Hunt' logo with 'Filter by platforms..' placeholder which is exactly meant for your problem. You can set your preferred programming challenges platforms there.

The size of the extension is not because of the libraries. It's basically because of the images of all the challenges platforms. We can get rid of them by asynchronously loading them from the web. But that's a little low in our priority list. We will try and see other ways of reducing the size.

Thanks a lot for your feedback!

**Comment by GaurangTandon** [comment]
Glad to get a quick response! No. Don't load the images asynchronously. That won't be good. I recommend, but I hope you have already done that, to use an image minifier (and a minifier for the scripts and css as well, like uglifyjs and cssminifier) to reduce the size a bit.

Have a good day!

**Comment by architv** [comment]
Sure, I think we could use the image minifier. Thanks a lot for the suggestion!
By the way, did your problem resolve? Were you able to set preferences?

**Comment by GaurangTandon** [comment]
@architv Yes. That's great! It worked, except that it sets what sites are to be shown, instead of what sites are _not_ to be shown. I can manage with that, as I only need to set it once.
